apps/server/src/modules/school/api/school.uc.ts

```diff
@@ -16,17 +15,18 @@ export class SchoolUc {
 		private readonly schoolYearService: SchoolYearService
 	) {}
 
-	public async getSchool(schoolId: EntityId, userId: EntityId): Promise<SchoolResponse> {
+	public async getSchoolById(schoolId: EntityId, userId: EntityId): Promise<SchoolResponse> {
 		const [school, user, schoolYears] = await Promise.all([
-			this.schoolService.getSchool(schoolId),
+			this.schoolService.getSchoolById(schoolId),
 			this.authorizationService.getUserWithPermissions(userId),
 			this.schoolYearService.getAllSchoolYears(),
 		]);
 
 		const authContext = AuthorizationContextBuilder.read([]);
 		this.authorizationService.checkPermission(user, school, authContext);
 
-		const yearsResponse = YearsResponseMapper.mapToResponse(school, schoolYears);
+		const { activeYear, lastYear, nextYear } = SchoolYearUtils.computeActiveAndLastAndNextYear(school, schoolYears);
```

Comment:
Warum utils und nicht service als Name?

Reply:
Ich dachte, weil es nur synchrone Methoden sind, die was aus den Inputs berechnen. "Service" ist ja bei uns (und auch sonst, wie ich es kenne) eine Klasse, die asynchrone Operationen durchführt z.B. in der Datenbank.